tests/unit_tests/caching_test.py

```diff
@@ -243,31 +245,32 @@ def test_graph_to_fun_pair_list(graph):
         graph.graph_to_fun_pair_list("main_function", "main_function")
     for side in [0, 1]:
         assert {obj[side].name for obj in objects_to_compare} == {
-            "do_check", "MACRO", "struct file"}
+            "do_check", "MACRO", "struct_file"}
         do_check = [obj[side] for obj in objects_to_compare
                     if obj[side].name == "do_check"][0]
         macro = [obj[side] for obj in objects_to_compare
                  if obj[side].name == "MACRO"][0]
         struct_file = [obj[side] for obj in objects_to_compare
-                       if obj[side].name == "struct file"][0]
+                       if obj[side].name == "struct_file"][0]
         assert do_check.filename == "app/main.c"
         assert do_check.line == 105
-        assert do_check.callstack == "do_check at app/main.c:58"
+        assert str(do_check.callstack) == "do_check at app/main.c:58"
         assert do_check.diff_kind == "function"
         assert do_check.covered
         assert macro.filename is None
         assert macro.line is None
-        assert macro.callstack == ("do_check at app/main.c:58\n"
-                                   "_MACRO at test.c:1\n"
-                                   "__MACRO at test.c:2\n"
-                                   "___MACRO at test.c:3")
+        assert str(macro.callstack) == ("do_check at app/main.c:58\n"
+                                        "_MACRO (macro) at test.c:1\n"
+                                        "__MACRO (macro) at test.c:2\n"
+                                        "___MACRO (macro) at test.c:3")
         assert macro.diff_kind == "syntactic"
         assert not macro.covered
         assert struct_file.filename == "include/file.h"
         assert struct_file.line == 121
-        assert struct_file.callstack == ("do_check at app/main.c:58\n"
-                                         "struct file (type) at "
-                                         "include/file.h:121")
+        assert str(struct_file.callstack) == (
+            "do_check at app/main.c:58\n"
+            "struct_file (type) at "
+            "include/file.h:121")
```

Comment:
should fit one line now:
```suggestion
            "struct_file (type) at include/file.h:121")
```

Reply:
Thanks, edited.